fastlane/lib/fastlane/actions/app_store_build_info.rb

```diff
@@ -0,0 +1,195 @@
+          FastlaneCore::ConfigItem.new(key: :version,
+                                       env_name: "LATEST_VERSION",
+                                       description: "The version number whose latest build number we want",
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :platform,
+                                       short_option: "-j",
+                                       env_name: "APPSTORE_PLATFORM",
+                                       description: "The platform to use (optional)",
+                                       optional: true,
+                                       is_string: true,
+                                       default_value: "ios",
+                                       verify_block: proc do |value|
+                                         UI.user_error!("The platform can only be ios, appletvos, or osx") unless %('ios', 'appletvos', 'osx').include?(value)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :team_name,
+                                       short_option: "-e",
+                                       env_name: "LATEST_TESTFLIGHT_BUILD_NUMBER_TEAM_NAME",
+                                       description: "The name of your iTunes Connect team if you're in multiple teams",
+                                       optional: true,
+                                       code_gen_sensitive: true,
+                                       default_value: CredentialsManager::AppfileConfig.try_fetch_value(:itc_team_name),
+                                       default_value_dynamic: true,
+                                       verify_block: proc do |value|
+                                         ENV["FASTLANE_ITC_TEAM_NAME"] = value.to_s
+                                       end)
+        ]
+      end
+
+      def self.output
+        [
+          ['LATEST_BUILD_NUMBER', 'The latest build number of either live or testflight version'],
+          ['LATEST_VERSION_NUMBER', 'The latest version number of either live or testflight version']
+        ]
+      end
+
+      def self.details
+        "Returns the current build number and version number of either the live or testflight version - it is useful for getting the build_number of the current or ready-for-sale app version, and it also works on non-live testflight version. If you need to handle more build-trains please see `latest_testflight_build_number`"
+      end
+
+      def self.example_code
+        [
+          'app_store_build_info',
+          'app_store_build_info(
+            app_identifier: "app.identifier",
+            username: "user@host.com"
+          )',
+          'app_store_build_info(
+            live: false,
+            app_identifier: "app.identifier",
+            version: "1.2.9"
+          )'
+        ]
+      end
+
+      def self.authors
+        ["rishabhtayal"]
```

Comment:
Can you included `hjanuschka` in here as well? Since that is where this code was originally derived from? 😊